server/routes/referrals/needsAndRequirementsPresenter.ts

```diff
@@ -1,5 +1,6 @@
 import { DraftReferral } from '../../services/interventionsService'
 import { FormValidationError } from '../../utils/formValidationError'
+import { SummaryListItem } from '../../utils/viewUtils'
```

Comment:
I think that the presenters should be written without reference to any view concepts / files. In general the presenter doesn't have any knowledge of the view. So I think best to put this in an independent file that's not part of something view-y, imported here and in `viewUtils`, like `summaryListPresenter.ts` or something.

Reply:
I've extracted into a util called `summaryList`, I thought that might be better than a fully-fledged presenter, as we seem to have specific patterns for those in this project.